### PR TITLE
fix(Plugin): add nodelay for limit-req plugin

### LINKED ISSUE #2015: sync with the latest schema from APISIX 2.7
The Apache APISIX 2.7 has been released, this PR https://github.com/apache/apisix-dashboard/pull/1999/files contains the differences from schema.json.

**TODO**
- [ ] Plugin: support all missing fields or newly added plugins in the Web
- [ ] API: support all missing fields in the Web

### PR DESCRIPTION
**Pull request by Baoyuantop**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

According to the latest `schema.json`, add `nodelay` options to `limit-req` plugin.

**Related issues**

fix #2015 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 1]
```diff
@@ -88,10 +95,21 @@ const LimitReq: React.FC<Props> = ({ form, schema }) => {
         initialValue={propertires.rejected_code.default}
         tooltip={formatMessage({ id: 'component.pluginForm.limit-req.rejected_code.tooltip' })}
       >
-        <InputNumber min={propertires.rejected_code.minimum} max={propertires.rejected_code.maximum} />
+        <InputNumber
+          min={propertires.rejected_code.minimum}
+          max={propertires.rejected_code.maximum}
+        />
+      </Form.Item>
+      <Form.Item
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/limit-req.tsx:103]: @Baoyuantop I notice this PR's main change is here, and other changes are made by your IDE or some other tools? They confuse me when I review your PR, please don't change them in this PR.

BTW, please test this new field, you could ask @guoqqqi for help.

Comment (Baoyuantop) [reply web/src/components/Plugin/UI/limit-req.tsx:103]: when I `commit` code, the `lint-staged`  is executed, then the code becomes like this.

Does it seem that too much code was not executed by `lint-staged` before?🤔

Comment (juzhiyuan) [reply web/src/components/Plugin/UI/limit-req.tsx:103]: maybe, because `husky` is introduced days ago, and don't have a global lint.

**Comment by liuxiran** [comment]
looking forward to your test case, thanks a lot

**Comment by Baoyuantop** [comment]
@liuxiran @guoqqqi

**Review by guoqqqi — APPROVED** [review]

**Comment by guoqqqi** [comment]
LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**